doc/man/man8/dmg.8.orig

```diff
@@ -0,0 +1,654 @@
+<<<<<<< HEAD
```

Comment:
file should be git rm?

Reply:
Oops, sorry. That was a temporary file, will remove.